app/src/main/java/com/framgia/forder/data/model/DataSuggest.java

```diff
@@ -0,0 +1,67 @@
+package com.framgia.forder.data.model;
+
+import android.os.Parcel;
+import com.arlib.floatingsearchview.suggestions.model.SearchSuggestion;
+import io.realm.RealmObject;
+
+/**
+ * Created by Age on 4/27/2017.
+ */
+
+public class DataSuggest extends RealmObject implements SearchSuggestion {
+    private static boolean mIsHistory = false;
+    private String mName;
+    private int mDomainId;
+
+    public DataSuggest() {
+    }
+
+    public DataSuggest(int domainId, String name) {
+        mDomainId = domainId;
+        mName = name;
+    }
+
+    public DataSuggest(String name) {
+        mName = name;
+    }
+
+    public static boolean getIsHistory() {
+        return mIsHistory;
+    }
+
+    public void setIsHistory(boolean isHistory) {
+        mIsHistory = isHistory;
```

Comment:
ERROR: [STYLE][ST_WRITE_TO_STATIC_FROM_INSTANCE_METHOD] Write to static field com.framgia.forder.data.model.DataSuggest.mIsHistory from instance method com.framgia.forder.data.model.DataSuggest.setIsHistory(boolean)
http://findbugs.sourceforge.net/bugDescriptions.html#ST_WRITE_TO_STATIC_FROM_INSTANCE_METHOD